app/views/admin_posts/show.html.erb

```diff
@@ -6,16 +6,20 @@
     <!--/descriptions-->
     <!--subnav-->
     <div class="navigation actions module" role="navigation">
-      <% if policy(@admin_post).can_post? %>
+      <% if logged_in_as_admin? %>
```

Comment:
Oh, dammit, _now_ that I merged this, I realized what was bugging me about this /o\ 

This should be `<% if policy(@admin_post).can_post? || (logged_in_as_admin? && @admin_post.find_all_comments.unreviewed_only.exists?) %>` to ensure we're not including an unnecessary heading and empty `ul` in cases where there are no unreviewed comments and you're logged in as an admin who can't post.

Do you want to do a pull request or should I?

Reply:
Visually there's no difference, but I guess there could be issues with screen readers. I have time to do another PR, but I think it should be
```rb
      <% if policy(@admin_post).edit? || policy(@admin_post).destroy? || (logged_in_as_admin? && @admin_post.find_all_comments.unreviewed_only.exists?) %>
```
just in case we ever have a role that can _post_ but not edit or delete. Either that, or I'd change the if checks for edit/destroy back to check `can_post?` too